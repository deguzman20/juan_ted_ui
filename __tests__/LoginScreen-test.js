// import 'react-native';
// import React from 'react';
// import { connect } from 'react-redux';
// import LoginScreen from '../src/screens/unauthorized/LoginScreen';
// import renderer from 'react-test-renderer';

// test('Login snapShot', () => {
//     const snap = renderer.create(
//         connect(<LoginScreen />)
//     ).toJSON();
// expect(snap).toMatchSnapshot();
// })