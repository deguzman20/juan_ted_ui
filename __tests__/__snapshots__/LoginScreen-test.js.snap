// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login snapShot 1`] = `null`;
